docs/reference/mapping.asciidoc

```diff
@@ -68,6 +68,14 @@ via the <<multi-fields>> parameter.
 [float]
 === Settings to prevent mappings explosion
 
+Defining too many fields in an index or having too many nested documents or too many fields that contain nested
```

Comment:
if you mean when using the nested type, nested documents are submitted together with their "parent" document. So, yes if they contain too many fields they lead to the same problem, but given that they are part of the main document their case is not different from "Defining too many fields in an index".